### ISSUE TITLE
CAM for YOLO-V8

### ISSUE THREAD

**Issue by ottato** [rigvedrs/YOLO-V11-CAM#20]
Can this be used for YOLOV5  as well?

**Comment by Billa-Man** [comment]
Yes, you can check [here](https://docs.ultralytics.com/models/) for the supported models and tasks maintained by ultralytics. Note that Ultralytics only supports Object Detection mode for YOLOv5 at the moment.
